src/test/java/com/easypost/Fixture.java

```diff
@@ -0,0 +1,281 @@
+
+        basicParcel.put("length", "10");
+        basicParcel.put("width", "8");
+        basicParcel.put("height", "4");
+        basicParcel.put("weight", "15.4");
+
+        return basicParcel;
+    }
+
+    /**
+     * Basic customs item that is reusable in all test coverage.
+     *
+     * @return A map that have the customs item info filled.
+     */
+    public static Map<String, Object> basicCustomsItem() {
+        Map<String, Object> customsItem = new HashMap<>();
+
+        customsItem.put("description", "Sweet shirts");
+        customsItem.put("quantity", "2");
+        customsItem.put("weight", "11");
+        customsItem.put("value", "23");
+        customsItem.put("hs_tariff_number", "654321");
+        customsItem.put("origin_country", "US");
+
+        return customsItem;
+    }
+
+    /**
+     * Basic customs Info that is reusable in all test coverage.
+     *
+     * @return A map that have the customs Info info filled.
+     */
+    public static Map<String, Object> basicCustomsInfo() {
+        Map<String, Object> customsInfo = new HashMap<>();
+
+        customsInfo.put("eel_pfc", "NOEEI 30.37(a)");
+        customsInfo.put("customs_certify", true);
+        customsInfo.put("customs_signer", "Steve Brule");
+        customsInfo.put("contents_type", "merchandise");
+        customsInfo.put("contents_explanation", "");
+        customsInfo.put("restriction_type", "none");
+        customsInfo.put("non_delivery_option", "return");
+        customsInfo.put("custom_items", basicCustomsItem());
+
+        return customsInfo;
+    }
+
+    /**
+     * Basic tax identifier that is reusable in all test coverage.
+     *
+     * @return A map that have the tax identifier info filled.
+     */
+    public static Map<String, Object> taxIdentifier() {
+        Map<String, Object> taxIdentifier = new HashMap<>();
+
+        taxIdentifier.put("entity", "SENDER");
+        taxIdentifier.put("tax_id_type", "IOSS");
+        taxIdentifier.put("tax_id", "12345");
+        taxIdentifier.put("issuing_country", "GB");
+
+        return taxIdentifier;
+    }
+
+    /**
+     * Basic shipment that is reusable in all test coverage.
+     *
+     * @return A map that have the shipment info filled.
+     */
+    public static Map<String, Object> basicShipment() {
+        Map<String, Object> basicShipment = new HashMap<>();
+
+        basicShipment.put("to_address", basicAddress());
+        basicShipment.put("from_address", basicAddress());
+        basicShipment.put("parcel", basicParcel());
+
+        return basicShipment;
+    }
+
+    /**
+     * Shipment that has all detail and is reusable in all test coverage.
+     *
+     * @return A map that have the shipment info filled in detail.
+     */
+    public static Map<String, Object> fullShipment() {
+        Map<String, Object> fullShipment = new HashMap<>();
+        Map<String, Object> options = new HashMap<>();
+
+        options.put("label_format", "PNG");
```

Comment:
The line comment `Must be PNG so we can convert to ZPL later` is missing here

Reply:
Nice catch.